net.sourceforge.roccat.roccat-tools.yml

```diff
@@ -0,0 +1,66 @@
+app-id: net.sourceforge.roccat.roccat-tools
+runtime: org.gnome.Platform
+sdk:  org.gnome.Sdk
+runtime-version: '44'
+
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=x11
+  - --device=all
+  - --device=dri
+  # necessary for audio feedback
+  # - --socket=pulseaudio
+  # - --socket=session-bus
+rename-icon: roccat
+
+modules:
+  - shared-modules/lua5.4/lua-5.4.json
+  - shared-modules/libcanberra/libcanberra.json
+  - shared-modules/gtk2/gtk2.json
+  - shared-modules/dbus-glib/dbus-glib.json
+
+  - name: libgaminggear
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_C_FLAGS=-isystem /usr/include/harfbuzz
+      - -DINSTALL_PKGCONFIGDIR=/app/lib/pkgconfig
+      - -DINSTALL_CMAKE_MODULESDIR=/app/lib/cmake
```

Comment:
I don't think these two lines are necessary.

Reply:
When I unset `DINSTALL_PKGCONFIGDIR` I get the following error:
```sh
-- Installing: /usr/share/pkgconfig/gaminggear-0.pc
CMake Error at configuration/cmake_install.cmake:54 (file):
  file INSTALL cannot copy file
  "/run/build/libgaminggear/configuration/gaminggear.pc" to
  "/usr/share/pkgconfig/gaminggear-0.pc": Read-only file system.
Call Stack (most recent call first):
  cmake_install.cmake:47 (include)
```

Similarly I get the following error when unsettings `DINSTALL_CMAKE_MODULESDIR`
```sh
-- Installing: /usr/share/cmake-3.25/Modules/FindGAMINGGEAR0.cmake
CMake Error at configuration/cmake_install.cmake:58 (file):
  file INSTALL cannot copy file
  "/run/build/libgaminggear/configuration/FindGAMINGGEAR.cmake" to
  "/usr/share/cmake-3.25/Modules/FindGAMINGGEAR0.cmake": Read-only file
  system.
Call Stack (most recent call first):
  cmake_install.cmake:47 (include)
```